source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -1249,16 +1249,9 @@ void Filter::sendImmediateResponse(const ImmediateResponse& response) {
   const auto mutate_headers = [this, &response](Http::ResponseHeaderMap& headers) {
     if (response.has_headers()) {
       absl::Status mut_status;
-      if (Runtime::runtimeFeatureEnabled(
-              "envoy.reloadable_features.immediate_response_use_filter_mutation_rule")) {
-        mut_status = MutationUtils::applyHeaderMutations(response.headers(), headers, false,
-                                                         config().mutationChecker(),
-                                                         stats_.rejected_header_mutations_);
-      } else {
-        mut_status = MutationUtils::applyHeaderMutations(
-            response.headers(), headers, false, config().immediateMutationChecker().checker(),
-            stats_.rejected_header_mutations_);
-      }
+      mut_status = MutationUtils::applyHeaderMutations(response.headers(), headers, false,
```

Comment:
nit: combine with previous line, and convert to `const`.

Reply:
done